nrzled/nrzled_spi_test.go

```diff
@@ -302,14 +311,17 @@ func TestSPI_Long(t *testing.T) {
 	if n, err := d.Write(toRGB(colors)); n != len(colors)*3 || err != nil {
 		t.Fatalf("%d %v", n, err)
 	}
-	expected := make([]byte, 12*o.NumPixels+3)
-	for i := 0; i < 12*o.NumPixels; i += 12 {
+	expected := make([]byte, 3+12*o.NumPixels+3)
+	expected[0] = 0x00
```

Comment:
Remove these, they are guaranteed to be zero.

Reply:
done. I made a comment, to explain why the loop starts with `i = 3`